project/Component/NefCommon/Models/RenderingURL.swift

```diff
@@ -0,0 +1,28 @@
+//  Copyright © 2020 The nef Authors.
+
+import Foundation
+import Bow
+import BowEffects
+
+public struct RenderingURL {
+    public let url: URL
+    public let title: String
+    public let escapedTitle: String
```

Comment:
I bet the `escapedTitle` is always computed in the same way from the `title`; therefore, could it be moved into a computed property to ensure we do not make a mistake when creating a `RenderingURL`?

Reply:
Great idea!

we can move `escapedTitle` to computer property. Logic to get the `title` could be different: it depends on if it is a page or playground...